app/scripts/views/confirm.js

```diff
@@ -56,6 +65,11 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
       this.navigate('signup');
     },
 
+    _openGmailTab: function () {
+      this._experiments.signal('openGmail', 'openGmailTab');
+      this.openGmailTab();
```

Comment:
Instead of calling an `openGmailTab` function to actually open the tab, you could do something like use the email address returned in the `context` function in the template and allow the [gmail tab to be opened from HTML](https://github.com/mozilla/fxa-content-server/pull/2943/files#diff-ecd846b040f06a700778d247aaccfdcaR16), and use the `_openGmailTab` function to call `this._experiments.signal`.

```js
   context: function () {
    var email = this.getAccount().get('email');
    var context = {
     email: email,
     safe_email: encodeURIComponent(email)
    };
   ...
   },
   ...
  _gmailTabOpened: function () {
    this._experiments.signal('openGmail', 'openGmailTab');
  },
...
```

This would allow you to remove the `openGmailTab` function from the experiment-mixin.

I'm wondering if the signalling can be moved to the mixin too, to be a bit more self contained. Perhaps via an `initialization` function and have the mixin declare its own `events` object.

```html
{{# experiments.isOpenGmailButtonVisible}}
  <div class="button-row">
    <a id="open-gmail" class="button" href="https://mail.google.com/mail/u/?authuser={{ safe_email }}" target="_blank">{{#t}}Open Gmail{{/t}}</a>
  </div>
{{/ experiments.isOpenGmailButtonVisible}}
```